ensime_shared/ensime.py

```diff
@@ -552,12 +552,7 @@ def handle_doc_uri(self, call_id, payload):
         if not self.en_format_source_id:
             self.log("handle_string_response: received doc path")
             port = self.ensime.http_port()
-            
-            url = payload["text"]
-            
-            if not url.startswith("http"): 
```

Comment:
He, unintentionally,  pushed a commit intended for another PR in this one. So this removed lines are the product of reverting that commit.
Now that #283 is merged, I think he should `rebase -i` to squash all commit but the revert one which should be discarded.